src/NServiceBus.Persistence.CosmosDB/Installer.cs

```diff
@@ -6,37 +6,50 @@
 using Installation;
 using Logging;
 using Microsoft.Azure.Cosmos;
+using Settings;
 
-class Installer(IProvideCosmosClient clientProvider, InstallerSettings settings)
+class Installer(IProvideCosmosClient clientProvider, IReadOnlySettings settings)
     : INeedToInstallSomething
 {
     public async Task Install(string identity, CancellationToken cancellationToken = default)
     {
-        if (settings == null || settings.Disabled)
+        InstallerSettings installerSettings = settings.Get<InstallerSettings>();
+        if (installerSettings.Disabled)
```

Comment:
Should we move this check out to the feature instead?

Reply:
Done